server/lib/routes/get-index.js

```diff
@@ -9,7 +18,27 @@ module.exports = function () {
   route.path = '/';
 
   route.process = function (req, res) {
-    res.render('index');
+    var time = Date.now();
+    var flowId = crypto.randomBytes(16).toString('hex');
+    // This event data is structured to be consistent with the auth server
+    // activity events. So yes, the format is different to the content server
+    // log lines. And yes, there is a mix of camelCase and under_score keys.
+    var activityEvent = {
+      device_id: null, //eslint-disable-line camelcase
+      event: 'flow.begin',
+      flow_id: flowId, //eslint-disable-line camelcase
+      flow_time: 0, //eslint-disable-line camelcase
+      new_user: false, //eslint-disable-line camelcase
+      time: time,
+      uid: null,
+      userAgent: req.headers['user-agent']
+    };
+    activityEventQueryParams.forEach(function (key) {
+      activityEvent[key] = req.query[key] || null;
+    });
+    // The data pipeline only reads from stderr.
+    process.stderr.write('activityEvent ' + JSON.stringify(activityEvent) + '\n');
+    res.render('index', { flowId: flowId, flowTime: time });
```

Comment:
:+1: We have a couple of flows that immediately render/return a result and then do further processing. Check out [POST /metrics](https://github.com/mozilla/fxa-content-server/blob/master/server/lib/routes/post-metrics.js#L25).